File: Example_Python/ReadJsonFiles.py
import json

with open("data.json","r") as f:
  data = f.readlines()
  res = json.loads("".join(data))

  # print( res["name"] )
  for n in range(len( res["dependencies"] ) ):
    publisher = res["dependencies"][n]["publisher"]
    name = res["dependencies"][n]["name"]
    version = res["dependencies"][n]["version"]
    print(publisher + "_" + name + "_" + version + ".app");

File: Example_Deno_javascript/ReadJsonFiles.js
export {};
/*
  ReadJsonFiles
  Reads a Jason struncture file and prints it contest to the screen

  Run:
    deno run --allow-read ReadJsonFiles.js

  Bygg:
    deno compile --allow-read ReadJson.js

  (C) 2022 Anders Persson
*/

let projectName = Deno.args[0];
projectName = ".";

const fullPath = projectName + String.raw`\data.json`;
const text = await Deno.readTextFile(fullPath);

const data = JSON.parse(text);
console.log(data.dependencies);
console.log(data.dependencies.length);

data.dependencies.forEach((id) => {
  console.log("Publisher: ",id.publisher);
  console.log("Name: ",id.name);
  console.log("Version: ", id.version);
  console.log("\n ------ \n");
});

Deno.exit(0);
